drivers/input/input_tsc_keys.c

```diff
@@ -0,0 +1,439 @@
+/*
+ * Copyright (c) 2024 Arif Balik <arifbalik@outlook.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <autoconf.h>
+#include <zephyr/irq.h>
+#include <zephyr/device.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/input/input.h>
+#include <zephyr/drivers/reset.h>
+#include <zephyr/sys/ring_buffer.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/clock_control/stm32_clock_control.h>
```

Comment:
that's an interesting sorting pattern :-)

Reply:
Hahaha, its just my ocd. Played with it a little bit, its grouped functionally now

Reply:
ah we don't have validation, nothing wrong with it, I normally just sort alphabetically, was mainly wondering if it randomly happened :-)